sql/hive-thriftserver/src/test/scala/org/apache/spark/sql/hive/thriftserver/CliSuite.scala

```diff
@@ -151,4 +155,33 @@ class CliSuite extends FunSuite with BeforeAndAfter with Logging {
         -> "hive_test"
     )
   }
+
+  test("Commands using SerDe provided in --jars") {
+    val jarFile =
+      "../hive/src/test/resources/hive-hcatalog-core-0.13.1.jar"
```

Comment:
This is because `hive-hcatalog-core-0.13.1.jar` is under another module (`hive`). Also, when a test case of a module is executed from SBT and Maven, the working directory is always the directory of that module.